osmaxx-py/osmaxx/contrib/auth/frontend_permissions.py

```diff
@@ -0,0 +1,51 @@
+from django.conf import settings
+from django.contrib.auth.decorators import login_required, user_passes_test
+from django.contrib.auth.models import Group
+from django.core.urlresolvers import reverse_lazy
+from django.utils.decorators import method_decorator
+
+FRONTEND_USER_GROUP = settings.OSMAXX_FRONTEND_USER_GROUP
+
+
+def frontend_access_required(function=None):
+    """
+    Decorator for views that checks that the user has the correct access rights,
+    redirecting to the information page if necessary.
+    """
+    access_denied_info_url = reverse_lazy('excerptexport:access_denied')
+    actual_decorator = user_passes_test(
+        user_in_osmaxx_group,
+        login_url=access_denied_info_url
+    )
+    if function:
+        return actual_decorator(function)
+    return actual_decorator
+
+
+def user_in_osmaxx_group(user):
```

Comment:
Maybe we should name these decorators more human readable like 
- is_user_in_osmaxx_group or
- is_user_in_osmaxx_group_check